Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG DOCKER_BUILDER_IMAGE=golang:1.16
+ARG DOCKER_BUILDER_IMAGE=golang:1.20
```

Comment:
0/5 while we are here, why not use go1.21?

Reply:
Agree, makes sense to bump to 1.21